crates/common/src/lib.rs

```diff
@@ -294,22 +294,23 @@ impl TransactionVersion {
         u128::from_be_bytes(lower.try_into().expect("slice should be the right length"))
     }
 
+    pub const fn with_query_version(mut self) -> Self {
+        self.0 .0[15] |= 0b0000_0001;
+        self
+    }
+
     pub const ZERO: Self = Self(H256::zero());
-    pub const ONE: Self = Self(H256(hex_literal::hex!(
-        "0000000000000000000000000000000000000000000000000000000000000001"
-    )));
-    pub const TWO: Self = Self(H256(hex_literal::hex!(
-        "0000000000000000000000000000000000000000000000000000000000000002"
-    )));
-    pub const ZERO_WITH_QUERY_VERSION: Self = Self(H256(hex_literal::hex!(
-        "0000000000000000000000000000000100000000000000000000000000000000"
-    )));
-    pub const ONE_WITH_QUERY_VERSION: Self = Self(H256(hex_literal::hex!(
-        "0000000000000000000000000000000100000000000000000000000000000001"
-    )));
-    pub const TWO_WITH_QUERY_VERSION: Self = Self(H256(hex_literal::hex!(
-        "0000000000000000000000000000000100000000000000000000000000000002"
-    )));
+    pub const ONE: Self = Self(H256([
+        0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0,
+        0, 1,
+    ]));
+    pub const TWO: Self = Self(H256([
+        0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0,
+        0, 2,
+    ]));
```

Comment:
Personal nit: considering we are setting them to literally `1`, and `2` I think the shorter `H256::from_low_u64_be(2)` is much more reasonable than trying to audit that 31/32 are zero + the number.
```suggestion
    pub const ONE: Self = Self(H256::from_low_u64_be(1));
    pub const TWO: Self = Self(H256::from_low_u64_be(2));
```

Reply:
Nevermind only saw the non-const comment after 😓 sorry

Reply:
`from_low_u64_be` is not const (unfortunately)